lib/database/activity.php

```diff
@@ -307,6 +307,10 @@ function RemoveComment($articleID, $commentID, $userID, $permissions)
 
 function addArticleComment($user, $articleType, $articleID, $commentPayload, $onBehalfOfUser = null)
 {
+    if (!in_array($articleType, \RA\ArticleType::values())) {
```

Comment:
Love this

Reply:
Thank! Anothers options are: 
* `\RA\ArticleType::isValid($articleType)`, or
* directly use https://github.com/myclabs/php-enum (but I don't know what is the meaning here about 3rd party lib)